docs/installation.rst

```diff
@@ -164,36 +151,51 @@ Installing using ``pip`` is more flexible than ``conda`` as it
 can support more versions of Python, and the locations of the
 various dependencies can be specified.
 
-Installing via ``pip`` is the recommended method when using the
-system provided Python installations.
+Msprime has a number of requirements which might or might not be
+installed already:
+
+* Python 3.4+
```

Comment:
We'll be going to 3.6+ for the release, so might as well get that up to date here.